docs/_cli_docs/docs.md

```diff
@@ -3347,6 +3463,4 @@ You can get help by either emailing `partners@zapier.com` or by [joining our dev
 
 ## Developing on the CLI
 
-See [CONTRIBUTING.md](https://github.com/zapier/zapier-platform/blob/master/CONTRIBUTING.md).
-
-{% endraw %}
```

Comment:
I noticed we still have `{% raw %}`, was removing`{% endraw %}` intentional?

Reply:
resolving - dupe to https://github.com/zapier/visual-builder/pull/401#discussion_r1087125298